test/transcoding/FPGAIVDepLoopAttr.ll

```diff
@@ -57,18 +57,21 @@
 ; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
 
 ; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
-; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
+; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM --allow-deprecated-dag-overlap
+; Note: --allow-deprecated-dag-overlap was enabled to allow indeterminate order of
```

Comment:
From [docs](https://llvm.org/docs/CommandGuide/FileCheck.html):

> This option is deprecated and is only provided for convenience as old tests are migrated to the new non-overlapping `CHECK-DAG`: implementation.

I wouldn't recommend use it in new tests. As a work-around, you can launch your test twice with different check-prefixes:

```
RUN: command-to-generate-llvm-ir | FileCheck %s --check-prefixes=common-prefix,prefix1
RUN: command-to-generate-llvm-ir | FileCheck %s --check-prefixes=common-prefix,prefix2

; COMMON-PREFIX-CHECK: capture EMB_IVDEP_DIM_2 and EMB_A_IDX_NODE_DIM_2
; PREFIX1-CHECK: ![[EMB_IVDEP_DIM_2]] = !{!"llvm.loop.parallel_access_indices",{{.*}} ![[EMB_A_IDX_NODE_DIM_2]]{{.*}}}
; PREFIX2-CHECK: ![[EMB_IVDEP_DIM_2]] = !{!"llvm.loop.parallel_access_indices",{{.*}} ![[EMB_B_IDX_NODE_DIM_2]]{{.*}}}
```

Reply:
Thanks for the suggestion. The new commit brings this approach in.